devlab/tests/config.py

```diff
@@ -201,7 +208,14 @@
     {'server': 'server2', 'image_name': 'asdasd'}
 ]
 
-# Cinder images to create/delete
+
+'''
+Cinder images to create/delete
+To write some date, use "write_to_file" parameter. Now only string could be
```

Comment:
Documentation about "write_to_file" option is kind of misleading, because it's not used in `cinder_volumes` instead it's used in `tenants['cinder_volumes']`. Not sure how to make it better, maybe point to `tenants['cinder_volumes']` in docstring?